src/startupscriptgenerator/src/common/logging.go

```diff
@@ -121,3 +129,13 @@ func (logger *Logger) Shutdown() {
 		// to complete, so let's just exit.
 	}
 }
+
+func (logger *Logger) StartupScriptRequested() {
```

Comment:
This log message can enable us to find out how many times Oryx run script was invoked